Makefile

```diff
@@ -108,8 +108,10 @@ clean: ## clean up binaries
 	@echo "🐳 $@"
 	@rm -f $(BINARIES)
 
-install: $(BINARIES) ## install binaries
-	@echo "🐳 $@"
+install: ## install binaries
+	@ls $(BINARIES) > /dev/null 2>&1 || \
```

Comment:
Agreed, but that was based on the old Makefile. 
You might be able to use something like this:
```
ifeq ($(shell uname -s),Darwin)
STAT           = stat -f '%c %N'
else
STAT           = stat -c '%Y %n'
endif
NEWEST_GO_FILE = $(shell find $(SRC_DIRS) -name \*.go -exec $(STAT) {} \; \
                   | sort -r | head -n 1 | sed "s/.* //")
```
and then use $(NEWEST_GO_FILE) in the dependency list.